src/Perpetuum/Zones/Terrains/Materials/Plants/PlantHandler.cs

```diff
@@ -21,10 +21,15 @@ public class PlantHandler : IPlantHandler
         public delegate IPlantHandler Factory(IZone zone);
 
         private const int AREA_SIZE = 32;
+        // The number of 32x32 cubes across the zone max width of 2048x2048
+        private const int TIME_SCALING_BASE = 64;
+        // Number of seconds in 8 hours
+        private const int EIGHT_HOURS = 28800;
+        private const int PLANT_REGEN_BASE_DELAY = EIGHT_HOURS / TIME_SCALING_BASE;
 
         private readonly IZone _zone;
         private readonly TimeSpan _natureSleepAmount = TimeSpan.FromSeconds(7);
```

Comment:
This value will also need to be updated.  It is hardcoded here as readonly from the assumption we are fixing.
Essentially this is storing a value that can not be manipulated by ScannerMode changes from issued commands.
(A fullspeed command essentially just runs with 0 delay, but then will reset to this "sleepAmount" once it is done or reset)